jenkins/get_ordered_test_list.py

```diff
@@ -0,0 +1,47 @@
+import click
+
```

Comment:
Can you add a general explanation in the beginning so that when someone comes across this in the future they know what we were using it for?